osquery/tables/events/freebsd/openbsm_events.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+
+#include <bsm/libbsm.h>
+
+#include <osquery/events.h>
+#include <osquery/logger.h>
+
+#include "osquery/events/darwin/openbsm.h"
+
+namespace osquery {
+
+namespace tables {
+extern long getUptime();
+}
+
+static inline void OpenBSM_AUT_SUBJECT32_EX(Row& r, const tokenstr_t& tok) {
```

Comment:
Maybe we should pull these out into a header file, or some OpenBSM utils. We now have this function duplicated twice and parsing for more tokens would need to be copied into both files. Thoughts? I can handle the testing on macOS if you want to do that :)

Reply:
Yeah, I was kind of leaving the multi-platform deduplication decisions to one of the core team members, cause this isn't the only place we have that. TBH the symlink + couple ifdefs approach is pretty straightforward but I know people hate that. :)